dev/com.ibm.ws.security.audit.reader/resources/com/ibm/ws/security/audit/reader/resources/UtilityMessages.nlsprops

```diff
@@ -63,3 +63,7 @@ security.audit.ErrorLoadingKeystore=Error loading the keystore {0}. The password
 security.audit.MismatchingEncKeystores=The input value, {0}, for the keystore specified to contain the certificate used to decrypt audit records does not match the keystore, {1}, specified in the audit log.
 security.audit.MismatchingSigningKeystores=The input value, {0}, for the keystore specified to contain the certificate used to unsign audit records does not match the keystore, {1}, specified in the audit log.
 security.audit.CannotFindCertificate=The {0} certificate alias does not exist in the {1} keystore.
+
+# Do not translate "auditRecord"
+security.audit.FailedToValidateSignature=Failed to validate the signature of the auditRecord at index [{0}].
+security.audit.FailedToDecryptAuditRecord=Failed to decrypt the auditRecord at index [{0}].
```

Comment:
The quality checked flagged this message as missing words. A full sentence that you might use instead is:

The decryption of the auditRecord failed at index [{0}].

Reply:
updated